app/assets/javascripts/discourse/app/templates/components/search-result-entry.hbs

```diff
@@ -10,7 +10,7 @@
       {{track-selected selectedList=selected selectedId=post.topic class="bulk-select"}}
     {{/if}}
 
-    <a href={{post.url}} {{action "logClick" post.topic_id}} class="search-link" role="heading" aria-level="2">
+    <a href={{post.url}} {{action "logClick" post.topic_id}} class="search-link{{if post.topic.visited " visited"}}" role="heading" aria-level="2">
```

Comment:
What is this change doing?

Reply:
There was another bug in the report. This was it and it was kind of related to these improvements. I moved it to a different PR: https://github.com/discourse/discourse/pull/15994

Reply:
I see. Yeah best to extract that in its own PR/commit 👍